dashboard.yml

```diff
@@ -76,6 +76,7 @@
 - hosts: "{{ groups[mgr_group_name] | default(groups[mon_group_name]) | default(omit) }}"
   gather_facts: false
   become: true
+  serial: 1
```

Comment:
I think this is needed because the config keys applied below will be cluster-wide hence we need to set and overwrite them from each node for the "dashboards update" command to work, is this correct?

Reply:
That's correct, I'm trying to translate in Ansible "words" the following:
```
>> for index in $(seq 1 3); do \
echo "set grafana-url to the instance deployed in controller-$index";
echo "set prometheus-host to the instance deployed in controller-$index";
echo "set alaertmanager-host to the instance deployed in controller-$index";
echo "inject the dashboard layouts to the current grafana node (deployed in controller-$index)";
echo "--------------------------------------"
done
```
where `controller` is the node where we're deploying the dashboard. For an external ceph context we can easily make `s/controller/mgr`.

Actually if you look at the templates [1] [2] [3] the endpoints are configured using `grafana_server_addr` fact, hence we can/should have an additional task to `unset` the endpoints in the mgr since they're not used (sounds like the data are coming from node-exporter and ceph CLI), but this is my understanding, maybe @b-ranto can help confirming this.


[1] https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-grafana/templates/datasources-ceph-dashboard.yml.j2
[2] https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-prometheus/templates/prometheus.yml.j2
[3] https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-prometheus/templates/alertmanager.yml.j2